src/gauravk/Assignment_5_Mall/Mall.java

```diff
@@ -0,0 +1,97 @@
+package gauravk.Assignment_5_Mall;
+
+import java.util.Scanner;
+
+/*
+ * 1. Create a class named as Mall, which contains below methods.
+a. itemAddedToCart()
+b. customerName()
+c. paymentDoneUsing()
+i. Visa Card ii. Cash
+d. displayCutomerInfo()
+e. needBill()
+f. haveCarryBag()
+g. decideSection()
+i.e. displayCustomerInfo() should print on console : Harsh buys 12 items and payment done using visa card in section 1B.
+Needbill - true
+Required CarryBag - false
+Expectations:
+� If items are less than 10 and payment done using cash, then send customer to section 1A
+� If items are more than 10 and payment done using visa card, then send customer to section 1B
+� If items are more than 10 and payment done using cash, then send customer to section 1C
+� If items are less than 10 and payment done using visa card, then send customer to section 1D
+Hint: itemAddedToCart method have int parameter, paymentDoneUsing() method have String parameter, 
+needBill() have Boolean parameter & haveCarryBag have Boolean parameter
+ */
+public class Mall {
+	int totalItem = 20;
+	int itemAdded = 0;
+	String custName;
+	String [] modeOfPayment = {"Cash","Visa"};
+	String selectedPaymentMode;
+	String section;
+	//String [] section = {"1A","1B","1C","1D"};
+	Scanner sc = new Scanner(System.in);
+	
+	void itemAddedToCart(int itemAdd) {
+		if(itemAdd<=totalItem) {
+			itemAdded = itemAdd;
+			System.out.println("How would you like to buy? (Cash or Visa)");
+			String s = sc.next();
+			paymentDoneUsing(s);
+		}
+		else System.out.println("Sorry the amount is higher than stock we have. We cannot fulfill your demand, unless you want to change your demand. Total stock is "+totalItem);
+	}
+	
+	void customerName() {
+		System.out.println("Please enter customer's name: ");
+		custName = sc.next();
```

Comment:
Scanner class should be in main method, as we will take all the input from the customer and then call respective method todo operation.

Reply:
Is it always necessary to use Scanner class in main method? What could be the reason for not using Scanner class in connected methods. Please explain.

Reply:
in real time we never use Scanner class, but for interview perspective we should know this class.
it's better use take input from entry point.

once we take all the input then we can execute our logic because ideally user will input after giving all the information then we expecting our requirement to fulfill.